### PR TITLE
Amend to fix potential Commit issue when unloading browser in SCORM 2004

### PR DESCRIPTION
**Pull request by danielghost**
https://github.com/adaptlearning/adapt_framework/issues/2222

### CODE REVIEW

**Comment by moloko** [comment]
hey @danielghost could you bring in the latest version of of SCORM_API_wrapper.js rather than just this one change? It will require a change to js/scorm/wrapper.js, just to change the module definition to:
```js
define ([
    'libraries/SCORM_API_wrapper'
], function(pipwerks) {
```

**Comment by danielghost** [comment]
@moloko - I did see the module definition changes but wasn't sure if it would need more thorough testing first before we introduce, whereas we know the other change won't affect anything else?

**Comment by moloko** [comment]
@danielghost I did give it a quick test already, it seems fine.

**Comment by moloko** [comment]
@danielghost do you want me to pick this up?

File: libraries/SCORM_API_wrapper.js [hunk 1]
```diff
@@ -24,820 +24,901 @@ further modified by Philip Hutchison
...
-                    switch(scorm.version){
-                        case "1.2" : success = scorm.set("cmi.core.exit", "logout"); break;
-                        case "2004": success = scorm.set("cmi.exit", "normal"); break;
                     }
 
                 }
 
-            }
+                //Ensure we persist the data for 1.2 - not required for 2004 where an implicit commit is applied during the Terminate
+                success = (scorm.version === "1.2") ? scorm.save() : true;
```
Comment (moloko) [inline libraries/SCORM_API_wrapper.js]: If it helps anyone to review, this is really the main change to this file.

The other big change is that the entire file has been converted into a module, you can see that change in the original repo [here](https://github.com/pipwerks/scorm-api-wrapper/commit/82e455b4032ee08febf64d2fa2bf1aacaebaa446). It was this change which necessitated a minor amend to wrapper.js

**Pull request merged**